src/app/shared/services/pages/products.service.ts

```diff
@@ -16,4 +16,7 @@ export class ProductsService {
   getItems(){
     return this.http.get("http://localhost:3000/api/product/get-products");
   }
+  getcategories(){
+    return this.http.get("http://localhost:3000/api/product/get-categories");
```

Comment:
no need to call it here just declare in environment folder and call it here in it or in service